### PR TITLE
Arcam FMJ 1.0.1

### PR DESCRIPTION
**Pull request by elupus**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/82487

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/arcam_fmj.markdown [hunk 1]
```diff
@@ -20,17 +20,26 @@ The `arcam_fmj` integration allows you to control [Arcam FMJ Receivers](https://
...
+- AVR 21
+- AVR 20
+- AVR 11
+- AVR 10
 - AV 860
-- SA 10
-- SA 20
-- SA 30
+- AV 41
+- AV 40
```
Comment (elupus) [inline source/_integrations/arcam_fmj.markdown]: A bit longer change than expected since i sorted the list.

**Review by MartinHjelmare — DISMISSED** [review]

File: source/_integrations/arcam_fmj.markdown (hunk 1 above)
Comment (MartinHjelmare) [reply source/_integrations/arcam_fmj.markdown]: Shouldn't we sort them 🔡?

**Review by elupus — COMMENTED** [review]
Adjust sorting... err no. that edit on github online didn't work.

**Review by MartinHjelmare — APPROVED** [review]
Thanks!

**Pull request merged**